.github/workflows/main.yaml

```diff
@@ -198,9 +200,19 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: artifacts/wasmer-macos-latest/wasmer-darwin-amd64.tar.gz
+          asset_path: artifacts/wasmer-macos-amd64/wasmer-darwin-amd64.tar.gz
           asset_name: wasmer-darwin-amd64.tar.gz
           asset_content_type: application/gzip
+      - name: Upload Release Asset Linux aarch64
```

Comment:
I'd like to be consistent about "arm" vs. "aarch64". I'm fine with either myself (I live my best life by pretending arm32 doesn't exist) but if you want to leave open the possibility of 32-bit arm in the future, change 'arm' to 'aarch64' throughout this PR.

Reply:
Done! All the remaining uses are required as that's the label that Github Actions uses (`ARM64`)